### PR TITLE
Remove snapshot schedule feature

### PR DESCRIPTION
**Pull request by remibergsma**
This removes these API calls:

```
createSnapshotPolicy
updateSnapshotPolicy
deleteSnapshotPolicies
listSnapshotPolicies
```

..and related stuff.

Tested manual snapshots and that still works fine.

### CODE REVIEW

**Comment by remibergsma** [comment]
These tables can be deleted. Can de done in this PR (but then we need an upgrade path) or later as it is only to clean unused tables. Let's discuss.

```
DROP TABLE IF EXISTS `cloud`.`snapshot_policy`;
DROP TABLE IF EXISTS `cloud`.`snapshot_policy_details`;
DROP TABLE IF EXISTS `cloud`.`snapshot_schedule`;
```

Update: as discussed will add them now in a separate commit.

File: cosmic-core/db-scripts/src/main/resources/db/schema-536to537-cleanup.sql [hunk 1]
```diff
@@ -2,3 +2,6 @@
 -- Schema cleanup from 5.3.6 to 5.3.7;
 --
 
+DROP TABLE IF EXISTS `cloud`.`snapshot_policy`;
```
Comment (eliasgomes) [inline cosmic-core/db-scripts/src/main/resources/db/schema-536to537-cleanup.sql:5]: I thought this was going to be included on a separate pull request with an upgrade path.
Is it correct to put it in a `cleanup` script? I think I'm missing something.

Comment (remibergsma) [reply cosmic-core/db-scripts/src/main/resources/db/schema-536to537-cleanup.sql:5]: It won't do much until the upgrade path is there. Since this file was already there (since KVM VM snapshotting feature) I also added it. This PR works fine without the tables being removed, and by committing it'll be automatically included in the next upgrade path. If you think it should go out, feel free to delete the commit.

Comment (eliasgomes) [reply cosmic-core/db-scripts/src/main/resources/db/schema-536to537-cleanup.sql:5]: Looks fine to me. But shouldn't it be on `schema-536to537.sql` instead of `schema-536to537-cleanup.sql`?

Comment (eliasgomes) [reply cosmic-core/db-scripts/src/main/resources/db/schema-536to537-cleanup.sql:5]: Discussed with @borisroman . Looks good!
